clients/src/test/java/org/apache/kafka/common/record/DefaultRecordBatchTest.java

```diff
@@ -153,6 +154,22 @@ public void testInvalidRecordSize() {
     }
 
     @Test(expected = InvalidRecordException.class)
+    public void testInvalidRecordSizeLow() {
```

Comment:
One of them is for record size too high, the other - for too low. I have clarified this by changing names appropriately.